### DISCUSSION TITLE
Login connection buttons do nothing and throw a console error

### DISCUSSION THREAD

**Discussion by mcolleague** [vercel/community#2771] [Bug Report]
### Page to Investigate

https://vercel.com/account/login-connections

## Steps to reproduce
- [ ] Login via any means
- [ ] Head to Settings > Login connections
- [ ] Click one of the Git provider buttons

## Screenshots
<details>
<summary>Trying to login via GitHub</summary>

![image (2)](https://github.com/vercel/community/assets/47757362/0bcfec7f-57ab-4b6c-8233-d6ab3ae1d109)
![image](https://github.com/vercel/community/assets/47757362/6d2cebe2-bbe9-4bd2-84f1-c43c185faaf8)

</details>
<details>
<summary>After logging in via email then clicking 'GitHub' login connection button</summary>

![image (1)](https://github.com/vercel/community/assets/47757362/50e67c35-9a90-4780-abce-10eb0bebc4e3)

</details>

**Comment by swarnava** [comment]
Hi,

We recently pushed a fix to a similar section. Are you still be able to reproduce this?

**Reply by mcolleague** [reply]
Can confirm buttons are working again, thanks!
